Lets Do This/Controllers/Login/LDTUserRegisterViewController.m

```diff
@@ -343,6 +362,13 @@ - (BOOL)validatePassword:(NSString *)candidate {
     return YES;
 }
 
+- (BOOL)validateNoEmojiPresent:(NSString *)candidate {
```

Comment:
Minor - but why do we need this function?  Feels like it'd be a little cleaner without it (just using `isIncludingEmjoi` for the checks like we do in other view controllers)